docs/src/debug.md

```diff
@@ -372,12 +287,10 @@ chromium.launch(new BrowserType.LaunchOptions() // or firefox, webkit
 
 ```python async
 await chromium.launch(headless=False, slow_mo=100) # or firefox, webkit
```

Comment:
this is like for playwright library, for the test runner they can use e.g. "--headed" which is the easiest.

Reply:
not really sure what you mean here. I didn't work on this section its the same as before I believe. So should I add a new section for --headed or ?

Reply:
lets follow-up: https://github.com/microsoft/playwright.dev/issues/681